src/utils/determine-correct-leafygreen-language.js

```diff
@@ -0,0 +1,23 @@
+import { Language } from '@leafygreen-ui/code';
+
+const SUPPORTED_LANGUAGES = new Set(Object.values(Language));
+
+export const determineCorrectLeafygreenLanguage = inputLang => {
+    let language = !!inputLang ? inputLang.toLowerCase() : 'none';
+    if (language === 'html' || language === 'toml') {
```

Comment:
Funny enough, the version of LG's code component we are using doesn't formally say these are supported but they are used successfully so I cased them out here.